Remotion/Development/UnitTesting.Compilation/Roslyn/RoslynCompiler.cs

```diff
@@ -378,6 +377,70 @@ private void Compile(CompilerParameters options, string compilerFullPath, string
             }
         }
 
+        // Adapted version of Executor.ExecWaitWithCapture that works in Linux.
+        private static int ExecuteCommand(
```

Comment:
I did not consider the details of this method during the review. I'm assuming you tested it and you're happy with the behavior excpet for the wait/exit logic.

Reply:
Running `Executor.ExecuteCommand` on Linux fails when the process is invoked. I used the source (https://source.dot.net/#System.CodeDom/System/CodeDom/Compiler/Executor.cs,30b6236af6f49d06,references) as template, remove all the unnecessary stuff and separated the file name and args when starting the process.